packages/cli/lang/en.lyaml

```diff
@@ -1420,9 +1420,7 @@ en:
         errorOccurred: "Error: {{ error }}"
         errorContext: "Context: {{ context }}"
         errorCause: "Cause: {{ cause }}"
-        systemErrorOccurred: "A system error has occurred: {{ errorMessage }}"
```

Comment:
`systemErrorOcurred` looks like it's used in `standardErrors.js`. Do we need to keep this around?

Reply:
Ah my bad. Yeah you're right 👌